ports/omniorb/portfile.cmake

```
@@ -0,0 +1,218 @@
+
+#string(APPEND VCPKG_C_FLAGS_DEBUG " -D_DEBUG") # for python to autolink the correct lib
+#string(APPEND VCPKG_CXX_FLAGS_DEBUG " -D_DEBUG")
+
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
+  set(z_vcpkg_org_linkage "${VCPKG_LIBRARY_LINKAGE}") 
+  # convoluted build system; shared builds requires 
+  # static library to create def file for symbol export
+  # tools seem to only dynamically link on windows due to make rules!
+  # zlib/zstd deps for ZIOP seem to not work on windows. At least configure
+  # won't run the required checks for some reasons.
+  set(VCPKG_LIBRARY_LINKAGE dynamic)
+  z_vcpkg_get_cmake_vars(cmake_vars_file)
+  include("${cmake_vars_file}")
+  if(VCPKG_BUILD_TYPE)
+    string(APPEND build_info "NoDebugBuild=1\n")
+  endif()
+  string(APPEND build_info "replace-with-per-config-text\n")
+  set(progs C_COMPILER CXX_COMPILER AR
+            LINKER RANLIB OBJDUMP MT
+            STRIP NM DLLTOOL RC_COMPILER)
+  list(TRANSFORM progs PREPEND "VCPKG_DETECTED_CMAKE_")
+  foreach(prog IN LISTS progs)
+      if(${prog})
+          set(path "${${prog}}")
+          unset(prog_found CACHE)
+          get_filename_component(${prog} "${${prog}}" NAME)
+          find_program(prog_found ${${prog}} PATHS ENV PATH NO_DEFAULT_PATH)
+          if(NOT path STREQUAL prog_found)
+              get_filename_component(path "${path}" DIRECTORY)
+              vcpkg_add_to_path(PREPEND ${path})
+          endif()
+      endif()
+  endforeach()
+  configure_file("${CMAKE_CURRENT_LIST_DIR}/vcpkg.mk" "${SOURCE_PATH}/mk/platforms/vcpkg.mk" @ONLY NEWLINE_STYLE UNIX)
+  file(GLOB_RECURSE wrappers "${SOURCE_PATH}/bin/x86_win32/*")
+  file(COPY ${wrappers} DESTINATION "${SOURCE_PATH}/bin")
+endif()
+
+vcpkg_configure_make(
+  SOURCE_PATH "${SOURCE_PATH}"
+  AUTOCONFIG
+  NO_WRAPPERS
+  COPY_SOURCE
+  OPTIONS
+    ${OPTIONS}
+)
+
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
+  vcpkg_replace_string("${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel//mk/platforms/vcpkg.mk" "replace-with-per-config-text" "NoDebugBuild=1")
+  if(NOT VCPKG_BUILD_TYPE)
+    vcpkg_replace_string("${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/mk/platforms/vcpkg.mk" "replace-with-per-config-text" "NoReleaseBuild=1\nBuildDebugBinary=1")
+    vcpkg_replace_string("${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/tool/omniidl/cxx/dir.mk" "python$(subst .,,$(PYVERSION)).lib" "python$(subst .,,$(PYVERSION))_d.lib")
+    vcpkg_replace_string("${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-dbg/src/tool/omniidl/cxx/dir.mk" "zlib.lib" "zlibd.lib")
+  endif()
+endif()
+
+vcpkg_install_make(
+  MAKEFILE "GNUmakefile"
+  ADD_BIN_TO_PATH
+)
+
+vcpkg_fixup_pkgconfig()
+
+if(VCPKG_TARGET_IS_WINDOWS AND NOT VCPKG_TARGET_IS_MINGW)
+  file(COPY "${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/lib/msvcstub.lib" DESTINATION "${CURRENT_PACKAGES_DIR}/lib")
+  file(GLOB all_libs "${CURRENT_PACKAGES_DIR}/lib/*.lib")
+  set(import_libs "${all_libs}")

```

```suggestion
  set(import_libs ${all_libs})
```

The intent appears to be to preserve the libs as separate list entries given the filter statement on the following line?

---

The intent is to make a 1:1 copy of the list without having to care about **spaces** in any element of the list. 

Also from the cmake maintainer guide:
`If an argument contains a variable expansion ${...}, it must be quoted.` 
and no the exception `Exception: a "splat" variable expansion...` does not apply here.

---

Ah, I see, thanks!